packages/swapper/src/types.ts

```diff
@@ -573,6 +574,7 @@ export type TradeStatus = {
   relayerTxHash?: string | undefined
   relayerExplorerTxLink?: string | undefined
   message: string | [string, InterpolationOptions] | undefined
+  actualBuyAmountCryptoBaseUnit?: string
```

Comment:
Start from here - carried from trade status endpoints, to swap (line diff above) and eventually consumed in `useActualBuyAmountCryptoPrecision`, pretty simple but diff just looks bigger than it is because of lint